server/bundles/io.cloudbeaver.service.security/src/io/cloudbeaver/service/security/CBEmbeddedSecurityController.java

```diff
@@ -2304,6 +2329,66 @@ protected SMTokens generateNewSessionToken(
         return generateNewSessionTokens(smSessionId, userId, authRole, dbCon);
     }
 
+    protected SMTokens kilAllExistsUserSessions(
+            @NotNull String smSessionId,
+            @NotNull String userId,
+            @Nullable String authRole,
+            @NotNull Connection dbCon
+    ) throws SQLException, DBException {
+        LocalDateTime currentTime = LocalDateTime.now();
+        List<String> smSessionsId = findActiveUserSessionIds(userId, currentTime);
+        deleteSessionsTokens(smSessionsId);
+        application.getEventController().addEvent(new WSUserEvent(smSessionsId, WSEventType.CLOSE_USER_SESSIONS));
+        return generateNewSessionTokens(smSessionId, userId, authRole, dbCon);
+    }
+
+    @NotNull
+    public List<String> findActiveUserSessionIds(
```

Comment:
use already exists findActiveUserSessions

Reply:
not included in CE